app/assets/stylesheets/components/_reorderable-list.scss

```diff
@@ -98,23 +98,21 @@
     }
   }
 
-  .app-c-reorderable-list__actions {
-    .govuk-form-group {
-      display: none;
-    }
+  .app-c-reorderable-list__actions .govuk-form-group {
+    display: none;
+  }
 
-    .gem-c-button {
-      display: inline-block;
-      margin-left: govuk-spacing(3);
-      width: 80px;
-    }
+  .app-c-reorderable-list__actions .gem-c-button {
```

Comment:
The nesting level on these is a sign we should try to refactor/simplify this component a bit in the near future.